mycore-base/src/test/java/org/mycore/services/http/MCRQueryParameterTest.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * This file is part of ***  M y C o R e  ***
+ * See http://www.mycore.de/ for details.
+ *
+ * MyCoRe is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MyCoRe is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MyCoRe.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.mycore.services.http;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+class MCRQueryParameterTest {
+    @Test
+    public void testOfEncodedString() {
+        MCRQueryParameter parameter = MCRQueryParameter.ofEncodedString("name=value");
+        assertEquals("name", parameter.name());
+        assertEquals("value", parameter.value());
+        parameter = MCRQueryParameter.ofEncodedString("name");
```

Comment:
additional test for edge case might be useful, probably:
```
parameter = MCRQueryParameter.ofEncodedString("name=");
assertEquals("name", parameter.name());
assertEquals("", parameter.value());
```

Reply:
added test for parsing "name=" in https://github.com/MyCoRe-Org/mycore/pull/2212/commits/669e4f6d271ff9a5718a9c3ae8db413811af7266